client/state/stored-cards/assembler.js

```diff
@@ -0,0 +1,11 @@
+export const createStoredCard = card => {
```

Comment:
May I suggest moving that logic to selector and storing data as it comes from the API?
@drewblaisdell do you remember the context related to this decision? I know we had a longer discussion on one of the PRs :)
I'm not sure if we have that stated explicitly in docs, but definitely it's worth checking: https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#selectors.